File: Backend/Financeiro.ConciliacaoWebAPI/JsonWebToken/Code/Permissao/PermissaoBLL.cs
using JsonWebToken.Code.Permissao;
using JsonWebToken.Model;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace JsonWebToken.Code
{
  public class PermissaoBLL
  {

    public List<PermissaoModel> SelectPermissao(List<string> lstNomGrpAcs)
    {
      try
      {
        return (new PermissaoDAL()).SelectPermissao(lstNomGrpAcs);
      }
      catch (Exception e)
      {
        throw e;
      }
    }
  }
}

File: Backend/Administracao.ConciliacaoAPI/Code/Permissao/PermissaoBLL.vb
Imports Administracao.ConciliacaoAPI.Model

Namespace Code.Permissao
  Public Class PermissaoBLL

    Public Function SelectPermissao(lstNomGrpAcs As List(Of String)) As List(Of PermissaoModel)
      Try
        Return (New PermissaoDAL()).SelectPermissao(lstNomGrpAcs)
      Catch e As Exception
        Throw e
      End Try
    End Function
  End Class
End Namespace
